app/pages/sections/service-recipients/contextCreator.js

```diff
@@ -1,8 +1,21 @@
 import manifest from './manifest.json';
 import { baseUrl } from '../../../config';
 
-export const getContext = ({ orderId }) => ({
-  ...manifest,
-  title: `${manifest.title} ${orderId}`,
-  backLinkHref: `${baseUrl}/organisation/${orderId}`,
-});
+export const getContext = ({ orderId, serviceRecipientsData = [] }) => {
+  const tableData = serviceRecipientsData.map(data => ({
```

Comment:
would be quite nice to extract this to a little helper function.

Reply:
we'll do it in the next pr as that's modifying the checkbox return